### ISSUE TITLE
It doesn't count my playtime.

### ISSUE THREAD

**Issue by makanm** [LeonardFiedrowicz/galaxy-integration-yuzu#7]
Is there a way to force the plugin to count my playtime?

I don't know is it normal or not, when i click on games gog doesn't detect them running to count the time i'v played!

**Comment by makanm** [comment]
Nevermind , just read the README.md, it doesn't track playtime yet.
So there will be no new updates to the plugin?

**Comment by hyoretsu** [comment]
@LeonardFiedrowicz @Pab10Suarez dunno if it's hard, but I got GOG Galaxy to track playtime when I simply linked Yuzu with `-f -g [game_location]`

**Comment by Pab10Suarez** [comment]
not sure why 

> @LeonardFiedrowicz @Pab10Suarez dunno if it's hard, but I got GOG Galaxy to track playtime when I simply linked Yuzu with `-f -g [game_location]`

not sure why you tag me but that's awesome !

**Comment by hyoretsu** [comment]
@makanm Glad to announce that it's fixed in [my improved fork](https://github.com/hyoretsu/galaxy-integration-switch).

**Comment by Pab10Suarez** [comment]
> 



> @makanm Glad to announce that it's fixed in [my improved fork](https://github.com/hyoretsu/galaxy-integration-switch).

that's awesome thank you, did you improved anything else? im asking because this plugin have some problems with game identify nsp files

**Comment by hyoretsu** [comment]
> > 
> 
> > @makanm Glad to announce that it's fixed in [my improved fork](https://github.com/hyoretsu/galaxy-integration-switch).
> 
> that's awesome thank you, did you improved anything else? im asking because this plugin have some problems with game identify nsp files

Yes, some of them are uploaded to this repo. (but the guy didn't create a new release)

It should identify any NSP name you may have, it now scans subfolders and XCI too. I also fixed a previous problem where some "bad" ROMs weren't being recognized.

Games also launch in fullscreen and it supports Ryujinx.

**Comment by Pab10Suarez** [comment]
> > > 
> > 
> > 
> > > @makanm Glad to announce that it's fixed in [my improved fork](https://github.com/hyoretsu/galaxy-integration-switch).
> > 
> > 
> > that's awesome thank you, did you improved anything else? im asking because this plugin have some problems with game identify nsp files
> 
> Yes, some of them are uploaded to this repo. (but the guy didn't create a new release)
> 
> It should identify any NSP name you may have, it now scans subfolders and XCI too. I also fixed a previous problem where some "bad" ROMs weren't being recognized.
> 
> Games also launch in fullscreen and it supports Ryujinx.

maybe we could talk to @Mixaill to put it in awesome gog galaxy https://github.com/Mixaill/awesome-gog-galaxy

**Comment by hyoretsu** [comment]
I already created a pull request to change the link in there

**Comment by makanm** [comment]
> @makanm Glad to announce that it's fixed in [my improved fork](https://github.com/hyoretsu/galaxy-integration-switch).

Wow i just saw this today! 
Awesome 😍

can u do the same thing with the cemu version?
It count playtime the first time but after restarting the pc or gog it stop counting playtime.
https://github.com/LeonardFiedrowicz/galaxy-integration-cemu/issues/30
